README.rst

```diff
@@ -1,131 +1,156 @@
 -  libzip >= 1.5.0 (>= 1.7.0 recommended)
 -  libzstd >= 1.4.0
+-  freetype2
 -  opusfile
 -  zlib
 
-Optional:
+Optional
+********
 
 -  OpenSSL (for a better SHA-256 implementation; used in shader cache)
 -  SPIRV-Cross >= 2019-03-22 (for OpenGL ES backends)
 -  libshaderc (for OpenGL ES backends)
+-  `ANGLE <https://github.com/google/angle>`__ (for ANGLE support, useful for Windows builds)
 -  GameMode headers (Linux only; for automatic `GameMode
    <https://github.com/FeralInteractive/gamemode>`__ integration)
 
-Build-only dependencies
-^^^^^^^^^^^^^^^^^^^^^^^
+Build-Time Dependenices
+_______________________
 
 -  Python >= 3.6
--  meson >= 0.53.0
--  [python-zstandard](https://github.com/indygreg/python-zstandard)
+-  `python-zstandard <https://github.com/indygreg/python-zstandard>`__ >= 0.11.1
+-  meson >= 0.56.0
+-  ``gcc`` or ``clang``
 
-Optional:
-
--  docutils (for documentation)
-
-Obtaining the source code
-^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Stable releases
-"""""""""""""""
-
-You can find the source tarballs at the
-`Releases <https://github.com/taisei-project/taisei/releases>`__ section on
-Github. **Do not** grab Github's auto-generated source archives, those do not
-contain the required submodules. This only applies for versions v1.3 and above.
+Obtaining Source Code
+^^^^^^^^^^^^^^^^^^^^^
 
-Latest code from git
-""""""""""""""""""""
+You can obtain the source code of the stable releases from the
+`Releases <https://github.com/taisei-project/taisei/releases>`__ page on
+GitHub, alongside the binaries. (Note that you **MUST** use the releases page,
+as the ``Download ZIP`` link on the main repo *does not* include all the code
+necessary to build the game.)
 
-If you cloned Taisei from git, make sure the submodules are initialized:
+You can also grab the latest code from git using the following commands:
 
 ::
 
-    git submodule init
-    git submodule update
+    git clone https://github.com/taisei-project/taisei
+    cd taisei
+    git submodule update --init --recursive
 
-This step needs to be done just once, and can be skipped if you specified the
-``--recursive`` or ``--recurse-submodules`` option when cloning.
+You should also run ``git submodule update`` whenever you pull in
+new code, checkout another branch, or perform any ``git`` actions. The ``pull``
+and ``checkout`` helper scripts can do that for you automatically.
 
-**Important:** You should also run ``git submodule update`` whenever you pull in
-new code, checkout another branch, etc. The ``pull`` and ``checkout`` helper
-scripts can do that for you automatically.
+**Important:** Again, make sure you download the source code from either the
+``Releases`` page *or* using ``git clone``. The ``Download ZIP`` link *will not
+work!*
 
-Compiling from source
+Compiling Source Code
 ^^^^^^^^^^^^^^^^^^^^^
 
-To build and install Taisei on \*nix, just follow these steps:
+Currently, we recommend building Taisei on a \*nix or macOS-based system.
```

Comment:
"POSIX-like" would cover them all (and even some obscure ones like Haiku — and yes, Taisei works there without modifications; no hardware acceleration though :smile:)